events/factories.py

```diff
@@ -0,0 +1,19 @@
+import factory
+from dateutil.tz import tzoffset
+from events.models import Event, Occurrence
+
+
+class EventFactory(factory.django.DjangoModelFactory):
+    name = factory.Faker("text", max_nb_chars=64)
+    duration = factory.Faker("random_int", max=1000)
+
+    class Meta:
+        model = Event
+
+
+class OccurrenceFactory(factory.django.DjangoModelFactory):
+    time = factory.Faker("date_time", tzinfo=tzoffset("UTC+2", 0))
```

Comment:
I think it would be better if the timezone was the one actually in used Finland, so that DST will be taken into account. IIRC it can be set with something like `tzinfo=pytz.timezone("Europe/Helsinki")`

Reply:
+1